drafts/developer-guide/pages/data-access-layer/index.adoc

```diff
@@ -0,0 +1,90 @@
+include::ROOT:partial$variables.adoc[]
+
+= Data Access Layer (DAL)
+
+The {PRODUCT_NAME} Data access Layer(DAL) extends and expands  Corteza’s capabilities to store and read data from primary and external databases.
+
+The data access layer helps us isolate( separation of concerns) our code from the presentation and business layers. 
+We use the data access layer to pull data from different data stores (database server, flat files, web services, whatever else).
+
+[TIP]
+====
+Wen changing the underlying database, for example, moving from one relational database like PostgreSQL to MySQL and then abstracting the underlying database is crucial.
+{PRODUCT_NAME} data access layer enables you to make the change with ease.
+====
+
+=== {PRODUCT_NAME} DAL Features.
```

Comment:
Consider rewording this to be more low-code developer friendly.
Is this section even necessary from the "I'm configuring a low-code app" perspective?

Reply:
Please explain